test/runtests.jl

```diff
@@ -44,7 +43,62 @@ facts("Utility functions") do
 end
 
 
-facts("Dam management") do
-    include("../examples/dam.jl")
-    solve_dams()
+facts("SDDP algorithm") do
+    solver = ClpSolver()
+
+    # SDDP's tolerance:
+    epsilon = .05
+    # maximum number of iterations:
+    max_iterations = 4
+    # number of scenarios in forward and backward pass:
+    n_scenarios = 10
+    # number of aleas:
+    n_aleas = 5
+    # number of stages:
+    n_stages = 2
```

Comment:
Instead of defining a number of stages, isn't it better to define a number of time step to avoid confusion between state and control ? For example, when there is just a control, a two stage problem is a problem with two decision. But if we add a stock and if we consider the final stock, the number of times steps is 3